### PR TITLE
Fix #69: Duplicate keys in presets

### LINKED ISSUE #69: Duplicate keys
A lot of your presets have duplicate keys.

Specifically, 
* Classic banks/IsBank/contact:email
* Classic banks/Liberty/atm
* Cosmetics and Chemistry/Ici Paris/brand:en
* Cosmetics and Chemistry/Voulez-Vous/brand:en
* Electronics/Alta/payment:alipay
* Electronics/Alta/payment:american_express
* Electronics/Alta/payment:apple_pay
* Electronics/Alta/payment:cash
* Electronics/Alta/payment:contactless
* Electronics/Alta/payment:credit_cards
* Electronics/Alta/payment:debit_cards
* Electronics/Alta/payment:diners_club
* Electronics/Alta/payment:discover_card
* Electronics/Alta/payment:google_pay
* Electronics/Alta/payment:jcb
* Electronics/Alta/payment:mastercard
* Electronics/Alta/payment:mir
* Electronics/Alta/payment:samsung_pay
* Electronics/Alta/payment:unionpay
* Electronics/Alta/payment:visa
* Electronics/Elit Electronics/payment:alipay
* Electronics/Elit Electronics/payment:american_express
* Electronics/Elit Electronics/payment:apple_pay
* Electronics/Elit Electronics/payment:cash
* Electronics/Elit Electronics/payment:contactless
* Electronics/Elit Electronics/payment:credit_cards
* Electronics/Elit Electronics/payment:debit_cards
* Electronics/Elit Electronics/payment:diners_club
* Electronics/Elit Electronics/payment:discover_card
* Electronics/Elit Electronics/payment:google_pay
* Electronics/Elit Electronics/payment:jcb
* Electronics/Elit Electronics/payment:mastercard
* Electronics/Elit Electronics/payment:mir
* Electronics/Elit Electronics/payment:samsung_pay
* Electronics/Elit Electronics/payment:unionpay
* Electronics/Elit Electronics/payment:visa
* Electronics/Zoommer/payment:alipay
* Electronics/Zoommer/payment:american_express
* Electronics/Zoommer/payment:apple_pay
* Electronics/Zoommer/payment:cash
* Electronics/Zoommer/payment:contactless
* Electronics/Zoommer/payment:credit_cards
* Electronics/Zoommer/payment:debit_cards
* Electronics/Zoommer/payment:diners_club
* Electronics/Zoommer/payment:discover_card
* Electronics/Zoommer/payment:google_pay
* Electronics/Zoommer/payment:jcb
* Electronics/Zoommer/payment:mastercard
* Electronics/Zoommer/payment:mir
* Electronics/Zoommer/payment:samsung_pay
* Electronics/Zoommer/payment:unionpay
* Electronics/Zoommer/payment:visa

### PR DESCRIPTION
**Pull request by tsmock**

### CODE REVIEW

File: ge.xml [hunk 1]
```diff
@@ -267,8 +267,6 @@
     <key key="payment:google_pay" value="yes" />
   </chunk>
   <chunk id="duty_free_assistance">
-    <reference ref="payment" />
-
```
Comment (tsmock) [inline ge.xml:271]: This is also done by `basic_shop`, and all instances of `duty_free_assistance` also use `basic_shop`.

**Review by empty-child — APPROVED** [review]

File: ge.xml [hunk 2]
```diff
@@ -1017,7 +1015,7 @@
           <key key="brand" value="ისი პარი" />
           <key key="brand:ka" value="ისი პარი" />
           <key key="brand:en" value="Ici Paris" />
-          <key key="brand:en" value="Иси Пари" />
```
Comment (tsmock) [inline ge.xml:1020]: This is presumed to be `brand:ru` due to L1024.

File: ge.xml [hunk 3]
```diff
@@ -1036,7 +1034,7 @@
           <key key="brand" value="ვულე ვუ" />
           <key key="brand:ka" value="ვულე ვუ" />
           <key key="brand:en" value="Voulez-Vous" />
-          <key key="brand:en" value="Вуле Ву" />
```
Comment (tsmock) [inline ge.xml:1039]: This is presumed to be `brand:ru` due to L1043.

File: ge.xml [hunk 4]
```diff
@@ -2361,7 +2359,6 @@
           <key key="name:ka" value="ლიბერთი" />
           <key key="name:en" value="Liberty" />
           <key key="name:ru" value="Либерти" />
-          <check key="atm" text="ATM" ka.text="ბანკომატი" ru.text="Банкомат" />
```
Comment (tsmock) [inline ge.xml:2364]: `bank_office` includes `atm`.

File: ge.xml [hunk 5]
```diff
@@ -2545,7 +2542,6 @@
           <key key="contact:facebook" value="https://www.facebook.com/IsbankGeorgia/" />
           <key key="contact:email" value="info@isbank.ge" />
           <key key="name:ka" value="იშბანკი" />
-          <key key="contact:email" value="info@rebank.ge" />
```
Comment (tsmock) [inline ge.xml:2548]: This is the line that is most likely to be wrong. L2546 actually has the bank name in it though.

**Pull request merged**